lang/py3/avro/schema.py

```diff
@@ -385,7 +385,7 @@ def NewWithDefaultNamespace(self, namespace):
     """
     return Names(names=self._names, default_namespace=namespace)
 
-  def GetName(self, name, namespace=None):
+  def _get_name(self, name, namespace=None):
```

Comment:
If I understand correctly, convention is for downstream users to consider methods that do not start with _ to be part of the public API and safe to use. But, all of the renames in the PR are breaking changes so we could just bump the version to indicate what happened.